### PR TITLE
feat: add duration as pressButton for tvOS

### PR DESCRIPTION
**Pull request by KazuCocoa**
Depends on https://github.com/appium/WebDriverAgent/pull/500

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -272,11 +272,11 @@ commands.getDevicePixelRatio = async function getDevicePixelRatio () {
 };
 
 commands.mobilePressButton = async function mobilePressButton (opts = {}) {
-  const {name} = opts;
+  const {name, duration} = opts;
   if (!name) {
     log.errorAndThrow('Button name is mandatory');
```
Comment (mykola-mokhnach) [inline lib/commands/general.js:277]: I would rather change it to InvalidArgumentError

File: README.md [hunk 2]
```diff
@@ -624,6 +624,7 @@ Emulates press action on the given physical device button.
 Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
 name | string | yes | The name of the button to be pressed. Supported button names for iOS-based devices are (case-insensitive): `home`, `volumeup`, `volumedown`. For tvOS-based devices (case-insensitive): `home`, `up`, `down`, `left`, `right`, `menu`, `playpause`, `select` | home
+duration | number | no | Duration in seconds for tvOS-based devices since Appium 1.22.0 | 10
```
Comment (mykola-mokhnach) [inline README.md:627]: in float seconds

Comment (mykola-mokhnach) [reply README.md:627]: You could also mention [mobile: performIoHidEvent](https://github.com/appium/appium-xcuitest-driver#mobile-performiohidevent) as a more universal API to perform press with duration on any supported device type

File: lib/commands/general.js [hunk 3]
```diff
@@ -271,18 +271,34 @@ commands.getDevicePixelRatio = async function getDevicePixelRatio () {
...
+ * @property {string} name - The name of the button to be pressed.
+ * @property {?number} durationSeconds - Duration in float seconds.
+ */
+
+/**
+ * Emulates press the given devive button name.
+ *
+ * @param {PressButtonOptions} opts
+ */
+
```
Comment (mykola-mokhnach) [inline lib/commands/general.js:285]: extra empty line here

File: lib/commands/general.js [hunk 4]
```diff
@@ -271,18 +271,34 @@ commands.getDevicePixelRatio = async function getDevicePixelRatio () {
...
 commands.mobilePressButton = async function mobilePressButton (opts = {}) {
-  const {name} = opts;
+  const {name, durationSeconds} = opts;
   if (!name) {
-    log.errorAndThrow('Button name is mandatory');
+    throw new errors.InvalidArgumentError('Button name is mandatory');
+  }
+  if (durationSeconds && !_.isNumber(durationSeconds)) {
+    throw new errors.InvalidArgumentError('durationSeconds should be number');
```
Comment (mykola-mokhnach) [inline lib/commands/general.js:292]: !_.isNil(durationSeconds) && !_.isNumber(durationSeconds)

Comment (mykola-mokhnach) [reply lib/commands/general.js:292]: sholud be a number

File: package.json [hunk 5]
```diff
@@ -50,7 +50,7 @@
     "appium-ios-simulator": "^3.28.0",
     "appium-remote-debugger": "^8.13.2",
     "appium-support": "^2.47.1",
-    "appium-webdriveragent": "^3.8.0",
+    "appium-webdriveragent": "^3.9.1",
```
Comment (mykola-mokhnach) [inline package.json:53]: Are you sure it is 3.9.1? I saw the latest is [3.8.1](https://github.com/appium/WebDriverAgent/releases/tag/v3.8.1) though

Comment (KazuCocoa) [reply package.json:53]: Yes. I pushed 3.9.0 before but the version in the package.json was back to 3.8.0 accidentally. Then, I pushed a patch as 3.8.1. After noticed it, I repushed 3.9.1...

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**